packages/core/src/editor/actions.ts

```diff
@@ -60,8 +57,11 @@ export const Actions = (
      */
     add(nodes: Node[] | Node, parentId: NodeId) {
       const parent = getParentAndValidate(parentId);
-      // reset the parent node ids
-      parent.data.nodes = [];
+
+      if (!parent.data.nodes) {
```

Comment:
I'm not resetting it but rather I'm creating the property because it doesn't exist yet.  This is because not all Nodes have child Nodes initially, so they don't necessarily have the `data.nodes` property specified. So what I did here was to initialised the property if it doesn't already exist. The same thing with `_childCanvas`.

Reply:
👍